### DISCUSSION TITLE
Z-Offset Issue

### DISCUSSION THREAD

**Discussion by Terminating12** [OpenNeptune3D/OpenNept4une#190] [Q&A]
I´m having three problems after installing OpenNeptune. 

First and most important, the CALIBRATE_PROBE_Z_OFFSET doesn't work. I calibrate this and save. Yet it still prints exactly the same way too high from the bed. 

Second is that the filament sensor doesn´t stay on even if I turn it on in settings and use SAVE_CONFIG

Third, how do I turn the beep of after a print?

Thanks in advance

**Comment by halfmanbear** [comment]
The beeper can be turned off by clearing the contents of the end tune macro.

The filament sensor is always off untill a print starts.

Check the z offset value in the end gray are of your printer.cfg before and after CALIBRATE_PROBE_Z_OFFSET

https://github.com/OpenNeptune3D/OpenNept4une/wiki/Printer-Calibration-%E2%80%90-Klipper-&-OrcaSlicer#more-info-on-z_offset

**Reply by Terminating12** [reply]
Hi,

I have fixed the z-offset by setting it to 0.000 and then calibrating CALIBRATE_PROBE_Z_OFFSET. It works now. If I don´t set it to 0.000 first, then it is always is too high from the bed.
